types/youtube/index.d.ts

```diff
@@ -518,6 +535,11 @@ declare namespace YT
 		 * Comma separated list of video IDs to play after the URL path's video.
 		 */
 		playlist?: string;
+		
+		/**
+		 * Whether videos play inline or fullscreen in an HTML5 player on iOS. (currently by default, Fullscreen).
+		 */
+		playsinline?: PlaysInline;
```

Comment:
There really shouldn't be of the const enum type because this means that a user cannot just pass `0` or `1` like in plain JS code.